kyuubi-server/src/test/scala/org/apache/kyuubi/config/AllKyuubiConfiguration.scala

```diff
@@ -203,8 +203,50 @@ class AllKyuubiConfiguration extends KyuubiFunSuite {
     newOutput += ""
     newOutput += ("Please refer to the Spark official online documentation for" +
       " [SET Command](http://spark.apache.org/docs/latest/sql-ref-syntax-aux-conf-mgmt-set.html)")
+    newOutput += ""
+
+    newOutput += ("## Flink Configurations")
+    newOutput += ""
 
+    newOutput += ("### Via flink-conf.yaml")
     newOutput += ""
+    newOutput += ("Setting them in `$FLINK_HOME/conf/flink-conf.yaml`" +
+      " supplies with default values for SQL engine application." +
+      " Available properties can be found at Flink official online documentation for" +
+      " [Flink Configurations]" +
+      "(https://nightlies.apache.org/flink/flink-docs-master/docs/deployment/config/)")
```

Comment:
Can we link a released version? The nightlies docs changing everytime.

Reply:
As the Flink version is updated too fast, I prefer to use nightlies/master docs.  And users can jump low version in master docs, but the inverse is not.

Reply:
use https://nightlies.apache.org/flink/flink-docs-stable/

Reply:
Thanks a lot, you are right.